dependencies.yaml

```diff
@@ -108,6 +108,10 @@ dependencies:
               cuda: "11.5"
             packages:
```

Comment:
Do we need to drop the 11.5 enty in the matrix since the above matrix for `x86_64`,`aarch64` + cuda version doesn't have 11.5 in it anymore

Reply:
For this `cudatoolkit:` section, we need to keep around any versions that we still test on.

So `11.5` should still stick around for now.